cmake/OpenCensusDeps.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(BUILD_TESTING)
```

Comment:
Why not use `externalproject_add()` for googletest too?

Reply:
I thought I was. If BUILD_TESTING is enabled, I do the usual dance starting on L19 with `configure_file`, which uses `cmake/googletest.CMakeLists.txt` which has the externalproject_add() call.